src/gpodder/util.py

```diff
@@ -595,6 +595,19 @@ def delete_file(filename):
         pass
 
 
+def is_html(text):
```

Comment:
> but found [\s\S]* equivalent to .* (isn't it?)

There is a difference when the element spans multiple lines: http://stackoverflow.com/a/35248940/1072626.  This may be important if the HTML was pretty printed.

An alternative option would be the dot-matches-all flag.

> r'<[a-z][a-z0-9]\*(\s.\*)?>'

Matches **<** followed by an alpha letter, followed by any number of alpha letter and numbers, optionally followed by a space then any number of characters on that line, followed by **>**

> r'<[a-z][\s\S]*>'

Matches **<** followed by an alpha letter, followed by any number of whitespace and non-whitespace characters (spanning lines), followed by **>**

> found no reason to not restrict the element name

I admit to being a little confused by the double negative, you "found reason to restrict the element name"?  Reading the regex, the difference seems to be in restricting to element name to (roughly) the ASCII alpha-numeric range.  I have been able to confirm that that is a restriction in Chrome on custom elements, but I have not easily verified whether HTML allows for Unicode in element names.  Especially if this can be confirmed, you may be right in that being a good restriction to add.


**tl;dr:**  Need to decide if elements should be allowed unicode characters, and decide whether to use [\s\S] or pass a flag to change dot behavior.